arbnode/transaction_streamer.go

```diff
@@ -205,6 +213,10 @@ func (s *TransactionStreamer) SetInboxReaders(inboxReader *InboxReader, delayedB
 	}
 	s.inboxReader = inboxReader
 	s.delayedBridge = delayedBridge
+
+	if s.syncMonitor != nil && inboxReader != nil && inboxReader.tracker != nil {
```

Comment:
s.inboxReader and s.inboxReader.tracker nil checks here are redundant, they will never be nil here, right?

Reply:
Removed the separate BroadcastSyncChecker.